src/encoder/encoderfdkaac.cpp

```diff
@@ -76,8 +76,18 @@ EncoderFdkAac::EncoderFdkAac(EncoderCallback* pCallback)
     libnames << bundlePath.absoluteFilePath();
 #endif
     libnames << QStringLiteral("fdk-aac");
+    // Although the line above should suffice, detection of the fdk-aac library
+    // does not work on Ubuntu 18.04 LTS and Ubuntu 20.04 LTS:
+    //
+    //     $ dpkg -L libfdk-aac1 | grep so
+    //     /usr/lib/x86_64-linux-gnu/libfdk-aac.so.1.0.0
+    //     /usr/lib/x86_64-linux-gnu/libfdk-aac.so.1
+    //
+    // As a workaround, we have to hardcode the filenames here.
+    libnames << QStringLiteral("libfdk-aac.so.1")
+             << QStringLiteral("libfdk-aac.so.2");
```

Comment:
Are you sure we need both of these? Is `libfdk-aac` not sufficient?

Reply:
No it isn't. No idea why.

Reply:
Since we support both version 1 and 2, we need both IMHO.